java/arcs/core/entity/EntityBase.kt

```diff
@@ -40,26 +40,26 @@ open class EntityBase(
     }
 
     /** Returns the value for the given singleton field. */
-    protected fun getSingletonValue(field: String): Any? = if (field in singletons) {
+    public fun getSingletonValue(field: String): Any? = if (field in singletons) {
```

Comment:
while we're here... use `getOrElse` pattern here as well?

Reply:
Another option is to use MapWithDefault, maybe Cameron has an opinion on this.

Reply:
I won't change it in the PR, if you come to a resolution it should just be a small change.